tdrs-frontend/cypress.config.js

```diff
@@ -6,10 +6,15 @@ module.exports = defineConfig({
   e2e: {
     baseUrl: 'http://localhost:3000',
     specPattern: '**/*.feature',
+    experimentalSessionAndOrigin: true,
```

Comment:
what's this do?

Reply:
This can be removed now. I needed it for some other feature I was testing. It turned out not to be necessary and I forgot to remove it.

Reply:
P.S., I have no clue how I made the above comment with my personal GH. I logged out and logged back in to make sure I am in the correct account.